app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -32,8 +32,7 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
       is DataForBackClickLoaded -> dataForHandlingBackLoaded(
```

Comment:
Can you reword this commit to fit in 1 line maybe?

"Change `PatientSummaryUpdate` to use count of recorded measurements on back click"